pkg/framework/api/common_state.go

```diff
@@ -51,6 +53,12 @@ const (
 	PodPropertyKey = "PodProperty"
 
 	AssignedNumasKey = "AssignedNumas"
+
+	PodSchedulingStageStateKey = "PodSchedulingStageState"
+
+	NotScheduledPodKeysByTemplateKey = "NotScheduledPodKeysByTemplate"
+
+	PostNodeCircleSchedulingFuncKey = "PostNodeCircleSchedulingFunc"
```

Comment:
nit: this is a temporary solution (for metrics) and should not be migrated to the open source repo

Reply:
fixed